core.js

```diff
@@ -1,10 +1,10 @@
-var https = require('https');
-var u = require('url');
-var buffer = require('buffer');
-const querystring = require('querystring');
+const https = require('https');
+const http = require('http');
+const u = require('url');
 
 function getAuthByHost (hostname) {
-    if (hostname === 'jenkins.whoop.com' || hostname === 'api.github.com') {
+    if (hostname === 'jenkins.whoop.com' || hostname === 'api.github.com' ||
+        hostname === '10.25.2.22') {
```

Comment:
Jenkins doesn't have a consistent internal IP address, so I think this should come out. If you wanted to look up CIDRs or netmasks you could accept 10.x.x.x, but I'd just keep it as a local-to-your-machine debugging hack.